### ISSUE TITLE
Accessibility: Part 2

### ISSUE THREAD

**Issue by lvuch** [rancher/dashboard#4393]
master: https://github.com/rancher/dashboard/issues/4324
13.
## Shared

- REPEATS IN OTHER PAGES] ![](RackMultipart20211014-4-19dhorn_html_71ae821e40c28e18.png) **image referenced** A few things: the element looks like a link but behaves like a button. Consider using a native button instead. If using the custom component is required, the element must be made focusable so that it can be accessed with keyboard.
- All table headers are interactive, however, they cannot be accessed via keyboard navigation ![](RackMultipart20211014-4-19dhorn_html_e53d79afa42f4b5a.png) **image referenced**
- [REPEATS IN OTHER PAGES] form elements missing labels: ![](RackMultipart20211014-4-19dhorn_html_bfbf2adaf8b316af.png) **image referenced**
- ![](RackMultipart20211014-4-19dhorn_html_3750cef20ad4b4c4.png) **image referenced** these icons are considered &quot;meaningful icons&quot; and are missing additional label. Meaningful icons are used to replace the usage of a text label and they require to always present an alternative sr-only span with a text description for SR.
- When the form is submitted and errors are encountered the list of errors are displayed at the bottom of the form. The W3C recommends placing the error list at the top, right before the \&lt;form\&gt; begins. On top of this, all input elements containing errors should be highlighted and marked with appropriate labels so that the user can easily identify them. [This applies to all other forms found on the application.] ![](RackMultipart20211014-4-19dhorn_html_c1447035b507f075.png) **image referenced**

## Light theme

- Use title attribute in iframe to describe its content
- Color contrast fails for the red star ![](RackMultipart20211014-4-19dhorn_html_4a0349f2fe032551.png) **image referenced** ![](RackMultipart20211014-4-19dhorn_html_6d830d586f9c6d22.png) **image referenced**

![](RackMultipart20211014-4-19dhorn_html_3750cef20ad4b4c4.png) **image referenced**

- Color contrast fails ![](RackMultipart20211014-4-19dhorn_html_6609f195b8c2b67a.png) **image referenced** ![](RackMultipart20211014-4-19dhorn_html_86d59fa2b3714f2e.png) **image referenced**

##

## Dark theme

- Color contrast fails in icon &amp; text

![](RackMultipart20211014-4-19dhorn_html_7791c4566f3411c7.png) **image referenced** ![](RackMultipart20211014-4-19dhorn_html_e83c1d15aadc3df8.png) **image referenced**

14.
## Shared

- `<Main> ` selector is duplicated due to inserting an iframe that already contains a \&lt;main\&gt; tag

15.
##

## Shared

- **Duplicated ID**![](RackMultipart20211014-4-19dhorn_html_6143a7b9b055e342.png) **image referenced**

##

## Dark theme

- The &#39;id&#39; &quot;#rules&quot; specified for the ARIA property &#39;aria-controls&#39; value is not valid (also applies to &quot;#related&quot;) ![](RackMultipart20211014-4-19dhorn_html_bd00ad295729484b.png) **image referenced**
